src/components/login-dialog/loginDialog.js

```diff
@@ -0,0 +1,31 @@
+import { Dialog } from 'primereact/dialog';
+import { Button } from 'primereact/button';
+import useLoginDialog from '@hooks/useLoginDialog';
+import { useRouter } from 'next/router';
+import style from './loginDialog.module.scss';
+
+export const LoginDialog = () => {
+  const loginDialog = useLoginDialog();
+  const router = useRouter();
+  const renderFoter = () => {
+    return (
+      <Button label="Ingresar" onClick={() => router.push('/users/sign_in')} />
+    );
+  };
+
+  return (
+    <Dialog
+      visible={loginDialog.displayLoginDialog}
+      className={style.dialog_login}
```

Comment:
it's seems weird to me that something here in front is using snake cale. (`dialog_login`)

Reply:
Done!